src/main/java/uk/gov/companieshouse/web/accounts/controller/cic/CICStepsToCompleteController.java

```diff
@@ -55,7 +55,8 @@ public String postStepsToComplete(@PathVariable String companyNumber,
 
             cicReportService.createCicReport(transactionId, companyAccountsId);
 
-            transactionService.createResumeLink(companyNumber, transactionId, companyAccountsId);
+            String resumeLink = "/company/" + companyNumber + "/transaction/" + transactionId + "/company-accounts/" + companyAccountsId + "/resume";
```

Comment:
We could construct this from a `UriTemplate`; On the class, define:
```
private static final UriTemplate RESUME_URI =
            new UriTemplate("/company/{companyNumber}/transaction/{transactionId}/company-accounts/{companyAccountsId}/resume");
```

And then to use this, call:
```
RESUME_URI.expand(companyNumber, transactionId, companyAccountsId).toString()
```